testsuite/features/core_ubuntu_salt.feature

```diff
@@ -1,48 +1,55 @@
 # Copyright (c) 2017-2018 SUSE LLC
```

Comment:
Please use 2017-2019 in the copyright

Also, rename to `core_min_ubuntu_salt.feature please` (name conventions are in the documentation).